test/prob/frechet/frechet_ccdf_log_test.hpp

```diff
@@ -69,9 +69,9 @@ class AgradCcdfLogFrechet : public AgradCcdfLogTest {
   stan::return_type_t<T_y, T_shape, T_scale> ccdf_log_function(
       const T_y& y, const T_shape& alpha, const T_scale& sigma, const T3&,
       const T4&, const T5&) {
-    using std::exp;
-    using std::log;
-    using std::pow;
+    using stan::math::exp;
+    using stan::math::log;
+    using stan::math::pow;
     return log(1.0 - exp(-pow(sigma / y, alpha)));
```

Comment:
We have a `log1m_exp` function built in and we should use that for numerical stability.   Or at least `log1p(-exp(...))`.

Reply:
Didn't really stop to think about what these are doing, but you are right. And if we are changing these files, it makes sense to clean them up as well.